content/agent/faq/supported-os-versions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Datadog Agent supported OS versions
+kind: faq
+disable_toc: true
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: Learn more about the Datadog Agent
+---
+
+### Agent 5+
+
+|OS| Supported versions|
+|:----|:----|
+|Debian x86_64 | Debian 7 (wheezy) and above (we do not support SysVinit)|
+|Ubuntu x86_64 | Ubuntu 12.04 and above|
```

Comment:
12.04 has been EOL'ed in Apr 2017, I don't know if agent5 still works well on it but we shouldn't support it officially. So let's say `14.04` and above instead.

Reply:
+1

Reply:
so, since `12.04` is based on debian wheezy there's no reason for the agent 5 to stop working there. I'm fine with keeping this now, we can revisit when wheezy is EOL'ed (end of May 2018)

Reply:
👍